### DISCUSSION TITLE
Slingbox Pro SB200 HD.

### DISCUSSION THREAD

**Discussion by avlad23** [GerryDazoo/Slinger#525] [General]
Hello! I have an Slingbox Pro SB200 HD whitch doesn't work anymore since last year. I read already a lot of informations about how can I turn it back on, but most of the informations are about the others type of Slingboxes (350-500). I am using the model SB200 because I am sending all analog TV channels from Romania here in USA and this was the perfect model to do that. Anyone can help me with the instuctions step by step how can I use it again, please? Because I saw, firstable I have to do the hard reset and to connected to the router and PC without internet connection, to create a new account (using the last Slingplayer app or the new Slinger Player app??); scan using the PC and app : wireshark to check the Slingbox port available and write that port into the router at Port Range Forwarding. And then can I connect the internet to send the TV channels on internet watching by Slinger Player app only? Because we have to avoid the Sling Company server; correct? How about the Remote Control? Because for the analog channels I never used the Slingbox sensor because in Romania I don't have a TV Box; the cable is coming from the Network company direct in my TV. On the old SlingPlayer I used just the software remote and worked good to change the channels. What do you think? I really appreciate every information! Thank you!

**Comment by GerryDazoo** [comment]
You should be good to go. Slinger supports a web-based remote control so you can change channels.
BTW, where did you read about Wireshark? That is definitely not required.

**Reply by avlad23** [reply]
I remember, I read something about Wireshark from reddit but maybe I misunderstood. 
In this case, how can I see which slingbox port is open in my slingbox? Or just I can pickup any port (like 5201 example) and this port should be similar in the Slingbox settings and Router Port forwarding? So, looks like nothing different than basic settings of the slingbox. What's the trick; how can I avoid that cloud server? I understand, that cloud server address isn't in the slingbox firmware and should be in the slingplayer app; correct? That is the reason why we should use now Slinger instead of SlingPlayer? 
So, the Slingplayer cannot be used any more with the Slingbox? 
At the first step when I will setup the slingbox, should I use Slingplayer app or the new app Slinger? 
I am litle bit confuse because I saw a lot of informations here but most of them are about the other type of slingboxes. 
For the Android phone, can I use the same app or I have to change it? 

Also I have an extra question :
The AverCaster AverMedia boxes : F210, F236-AF, F236, F239 and SE510 are working similar like the slingboxes? Did you try them?

Thank you and appreciate your help!

**Reply by GerryDazoo** [reply]
Run the server without an IP address configured in the config.ini and the software will probe your LAN for boxes and print out what it finds. If it only finds one box it will use it.
No idea about those other boxes.

**Reply by avlad23** [reply]
Just I ordered on ebay a new slingbox because I cannot fly back now to Europe to change the settings for the old Slingboxes. I order one here in USA to experiment all your advices. The seller just told me, the Slingbox I order, have not been in use; it is like new-unused. In this case, how can I have an email account and admin password? If I will do a factory reset, probably I can have the password : admin but also it is neccesary any email account? How can I have it? Thank you!

**Comment by avlad23** [comment]
Brick - How can I unbricked?
Just I received a brand new - unused SB200 Pro and I started to setup without any internet cable connected. I connected SB to Router and PC. I installed SP 1.5.1.342 and I did all the configurations, I have been able to setup the password. I read : FinderID, MAC, Hardware and firmware version, IP (192.168.3.237), Port (5001), Server IP (70.42.244.140) and Server Port (5678). I blocked the port in my Router (see the picture attached) and I connected the Internet cable but after few seconds the SB become bricked. I tried to do the hard reset (10-15 seconds) with the  internet cable disconnected and I can see the SB but I don't have access to the settings any more and I don't understand why? Anyone can help me, please? What I am doing wrong?Thank you!
![01 SP](https://user-images.githubusercontent.com/125126713/221906535-76cecfff-78ba-4241-b064-5973ee661326.jpg)
![02 SP](https://user-images.githubusercontent.com/125126713/221906574-a0321e2e-263e-4486-91f8-f43e457d5a27.jpg)

**Reply by flipper2222222** [reply]
the screenshot looks like it is showing port forwarding not port blocking.

**Reply by avlad23** [reply]
I tried to find on youtube how can I do "port blocking" but I didn't find nothing about that. Can you tell me step by step how can I do "port blocking", please? Also, do you have any ideea how can I do the hard reset of the slingbox? I tried to press that reset button for 10-15 seconds but still doesn't working. Thank you!

**Reply by flipper2222222** [reply]
go to the tp link site and download the manual for your router model to see if port blocking is supported, you might find it under parental controls menu
https://www.tp-link.com/au/support/faq/2386/

**Reply by avlad23** [reply]
Thank you for your reply! I have the same Archer C7 Router but what I understand the Control Parental will block the internet for the Slingbox; in this case how the slingbox can send the channels over the internet? ... should I keep also the Port Forwarding line ON or isn't necessary? For the time, I left it OFF (not setup) with unable access internet anytime; that is correct? In the "Blocklist" should I write something? Also when I did the setup, I didn't see which specific port I want to block? Do you know the answer of my questions or do you have other link with the details; please? Thank you and appreciate your time!

**Reply by flipper2222222** [reply]
everyone has different setups , my slingbox dosent need internet access so I blocked all access, some people only block the one port 5678 to stop the bricking
I keep my port forwarding to the slinger server not  to the slingbox. 
have you read the setup notes?
https://github.com/GerryDazoo/Slinger/blob/main/V4.0_release_notes.pdf

**Reply by avlad23** [reply]
Yes, I seen that V4.0 release note but because I am not a software developer is kind of difficult for me to understand every detail. I hope in the future someone to do a guide step by step (easy for everyone) to setup the slingboxes. Thank you for your help!

**Reply by flipper2222222** [reply]
is your slinger server and slingbox on the same network?

**Reply by avlad23** [reply]
I connected the Slingbox direct to the router (192.168.3.1) and then to the PC laptop (no internet cable). That means is on the same network? Or should I do a different configuration?

**Reply by flipper2222222** [reply]
sounds like you are using both on the same network, if you want to view the slingbox from the internet you will need the slinger port open on your router.

**Reply by avlad23** [reply]
My problem is because I can see the Slingbox (see the picture) but I don't have any Information available; that means the Slingbox is bricked? Because I tried to do hard reset many times and still I cannot turn on my Slingbox to have access to all informations. And if my Slingbox is bricked; can help me to run Slingbox_server.exe app? Because in my opinion I have to unbricked my Slingbox first and then I can run Slinbox_server.exe app; correct?

![03 SB](https://user-images.githubusercontent.com/125126713/222039861-689f8f19-a022-43ec-964c-62c48fa9e94d.jpg)

**Reply by flipper2222222** [reply]
I don't have any experience with bricked Slingboxes, one of the guys in North America can probably answer when they start for the day. good luck!

**Reply by avlad23** [reply]
I understand; no problem. Just let me know, please, how did you do the slinger port open in your router? Like in my case : Internal IP should be my slingbox IP 192.168.3.237; how about the port; should be Slingbox port 5001 or 5678 (server port) or 8080? What did you write on the Port Forwarding line to open the port of slinger in your router? Thank you!

**Reply by GerryDazoo** [reply]
No the IP address is NOT the address of the slingbox. It's the address of the host where you're running Slinger. In your case 192.168.3.205

**Comment by GerryDazoo** [comment]
That's a bit of a mess.
1) You only need to open the port on your router to Slinger if you intent to access your video remotely from the Internet. You would use the [Server] port number in your config.ini file (8080 is the default) but I recommend not using that.
2) The error you're seeing  "binding to broadcast address" means you've got more than one copy of the Slinger on your network. If you resolve that the code will scan your network and print out the info for any Slingboxes it finds.

**Reply by avlad23** [reply]
What I tried to do : I remove the SB200 and I replaced with another SB220, I didn't change anything in my Laptop and everything working good; I had access to the SB settings and information (see the picture). And then I removed SB220 and I install SB200 back and again I don't have access to the settings. So, why SB220 is working fine and SB200 doesn't work; should I change someting in my router settings?
What do you reffer when you said " you've got more than one copy of the Slinger on your network"? Should I change/delete someting in my router : Parental Control or Security or Port Forwarding? Thank you!
![04 SB220](https://user-images.githubusercontent.com/125126713/222222646-5a30bd86-8153-4169-b556-028bfcfb5b00.jpg)

**Reply by avlad23** [reply]
There is any way to eliminate completely the Server and to have direct video streaming from the Slingbox over the internet to the computer in different location?

**Reply by bradut** [reply]
Hi Vlad,

Yes, it is possible. 

Currently I am streaming from Romania to Canada/everywhere and watching on PC/Laptop via SP 2.x and 1.5.
Initially I used Slinger + VLC which worked great. Most likely will use Slinger again if I plan to stream on mobile devices.

SP 2.x gives you a decent resolution (1920x540) but may require you to enable Adobe Flash, while SP 1.5 is limited to 640x480.

I've blocked lots of ports not only the outgoing 5678, and port-forwarded whatever port (e.g. 45001) to 5001 which is being used by my SB#1

**Reply by flipper2222222** [reply]
@bradut  can you please tell us what ports your blocked and why you blocked a lot of ports not just 5678? thank you

**Reply by bradut** [reply]
@flipper2222222 

**Update: 03.03.2013**
I've just set up my second HD-PRO and I only needed to block the port 5678.
I've blocked source port 5678 and set the destination port = any port.



**Original answer:**
#1. "_what ports your blocked_ "

I've blocked these ports for TCP:
3074 | TCP |  
3490 | TCP |  
3724 | TCP |  
5678 | TCP |  
5680 | TCP |  
7752 | TCP | 

In my router, I've created two sets of rules for these ports:
1 blocked the destination, and left the source empty (= any port)
2 blocked the source, and left the destination empty   

The port 80 (and maybe 8080) was used by the SlingPlayer, so I've left it open.

#2 "_why you blocked a lot of ports not just 5678?_"
I know it sounds (and it is empirical), but only blocking 5678 still bricked my HD-PRO.

_Additionally:_
Also, as I mentioned in another post, the older version of SP 2.0, (older than 2.0.4.522), bricked my box. 
Maybe it was just me, but it happened a couple of times.

**Reply by GerryDazoo** [reply]
Whatever you're doing on your router to block outgoing TCP port 5678 is not correct otherwise your box would not keep getting bricked. 
Some router have the ability to block all outgoing traffic to the Internet based on MAC address, look for Parental Controls or something similar. Blocking all traffic for the slingbox to the internet will work too. That is how I'm setup.

**Reply by avlad23** [reply]
Bradut, I sent you a private message on Facebook. Are you able to help me to setup my Slingbox for direct streaming over the internet without the Server configuration, please? Thank you!

**Reply by bradut** [reply]
Got it :)

**Reply by bradut** [reply]
> Whatever you're doing on your router to block outgoing TCP port 5678 is not correct otherwise your box would not keep getting bricked. [...].

@GerryDazoo Thank you Gerry for your explanations,
- I've got bricked whenever I set up a port-range blocking, such as 3000:5000, 5010:9000 in two routers I used for two SlingBoxes.
- Indicating specifically the port(s) to block instead of ranges solved the problem.
- For my second SlingBox which I've setup two weeks ago, I only blocked port 5678. I've blocked it both ways - just to make sure.
- Today, after two weeks of working fine, that SlingBox has been bricked: In SP 1.5 It no longer accepts the password and in Slingbox Configuration -> Slingbox Settings it displays _Operation failed, Error: 0x92340070_. 

I suspect that there may be another "backdoor" port that has facilitated the bricking b/c the static IP of that SlingBox is the same and the port 5001 is still open.

I wish I could block all outgoing traffic, but I need to access that SlingBox from outside. 
What ports should I leave unblocked in this case (besides 80 and 5001) ?

PS: 
Not sure if this is related to my issue, but yesterday I've tried to access that Slingbox by name on my mobile phone, given that its original name was still displayed in the Slingbox directory.
It didn't work as, after introducing the new password, I've got the error message that was not compatible with Android. Maybe this attempt helped the Sling server to track my box and deactivate, ...I don't know.

**Reply by GerryDazoo** [reply]
Many people have scoured packet captures of the bricking process and have not seen any other ports involved. I suspect your Slingbox got a new DHCP IP address and your port blocking was based on LAN IP address.
When you unbrick setup a static IP for the slingbox on your LAN. I also recommend setting the default gateway address when you set this up to 127.0.0.1. This will also prevent your box from contacting the Internet.

**Reply by bradut** [reply]
Thank you @GerryDazoo Gerry.
The static IP didn't change, as I explained above, this is why I suspected that the poison came via another port.

One curious fact about this particular SlingBox is that, after it was bricked on Nov.9, it no longer accepted the previously assigned static IP, and assigned itself 192.168.0.254 . Never seen such a situation, but didn't have time to tinker with it, so assigned it its preferred static IP xxx.254. 

These are the settings that helped that SlingBox function in the past two weeks:
![image](https://user-images.githubusercontent.com/2496596/226490937-0834642f-8285-4d31-90ba-35903db64161.png)
[Just blocking only the outgoing (source) port 5678 should have been enough.]

I think that your recommendation about default gateway 127.0.0.1 is specific to the Slingbox_Server which I will start using soon together with SlingerPlayer, and will set up the gateway as you have recommended.
Currently, this SlingBox is located in a different country and I am accessing it via SlingPlayer 1.5 and 2.0.4.

**Reply by GerryDazoo** [reply]
If you're trying to access the slingbox remotely from the Internet then setting the default gateway on it to 127.0.0.1 will stop it from working. Sorry about that I hadn't realized that was your network configuration. Your posted config.ini shows the IP address on the LAN 192.168.1.x. Which makes me wonder how you're managing it on the LAN if it's IP is really 192.168.0.254 per your post.

**Reply by bradut** [reply]
Hi Gerry,
I am aware it may sound confusing:
- The 192.168.1.x is from my LAN, here in Canada.
- The 192.168.0.254 is from the Vodafone LAN in Romania. 
- I also have there an Orange LAN, 192.168.50.x which hosts my third SlingBox.

I've left there a laptop which is always on, connected to one LAN by wire, and to the other by WiFi. 
It helps me manage the networks, setup the SlingBoxes, and, in the near future, I will use it to run the Slingbox_Server. 
The laptop was very helpful yesterday when setting up the bricked box after reset did not work remotely from Canada, but worked from the remote laptop.

**Comment by avlad23** [comment]
Thanks to Bradut, he helped me out to setup my Slingbox Pro HD by remote and everything working good for now.
I still have two Slingboxes bricked : SB200 Pro and SB220 Tuner. I tried to do the hard reset up to 120 seconds but nothing helped. 
Also the admin password "admin" doesn't work any more. 
Does anyone figure out how SB200 Pro can be unbrick and let me know, please? 
Thank you!

**Reply by avlad23** [reply]
There is any way if I will update the firmware to unbrick the Slingbox? If I will try to edit the last version of the Firmware 1.4.20 by Hex Editor to a newer version like 1.4.50; can I unbrick the SB? Did someone tried that option?

**Comment by avlad23** [comment]
Important : Did anyone tried to modify the Slingbox Firmware (by Hex Editor) to Delete/erase the Port 5678 to Stop the Slingbox to send the signal to that bad port and to not be necessary any more to block this port in the router? That can be an option or not?

**Comment by kocoman1** [comment]
has anyone dump the parallel flash with hot air station etc? the original fw password is if you search for ".com" in hex editor.. but if some other important data is wiped. ?

**Comment by avlad23** [comment]
Since 4 months ago I have 3 SlingBoxes (SB200, SB240 and SB300 ProHD) working good by Direct connection to the original SlingPlayer (Windows11) because I have the same issue like Bradut with my Router is IPv6 and I cannot use the Slinger_Server for SlingerPlayer over the internet.
Does anyone know why the original SlingPlayer is working by Direct connection without using any Server? 
There is any way in the future to have a software Player app to work by direct connection without any server in between?
Or the original Slingplayer still has a server which is still running somewhere?

**Reply by GerryDazoo** [reply]
The "server" in SlingPlayer's case is embedded in the Player application. It talks to the Slingbox directly.
I can't test this myself but I believe it would be possible to setup a VPN connection over IPV6 into your home network and then tunnel the needed IPV4 connection inside the IPV6 connection. It also might be possible with just a reverse tunnel over an SSL IPV6  connection

**Reply by avlad23** [reply]
Thanks! Interesting how the original SlingPlayer working with the "server" embedded without to be necessary to do Port Forwarding. 
About IPv6 I understand there is a way to work by VPN; but not every VPN; because I have an account with NordVPN and I think this one doesn't work like that.
Maybe is much easy to try to talk to T-Mobile (the property of my modem) to see if they can disable my IPv6.
Do you know if there is any software program to breake the SlingPlayer for Android to have the source code?
I think can be interseting to see how they embedded the "server" and if we can change the application for direct connection.

**Reply by GerryDazoo** [reply]
It does do port forwarding (you just don't see it) by talking to your router using a protocol called UPnP.
I'm referring to setting up your own VPN server something like OpenVPN. a VPN service like NordVPN will not work for this.
Your problem isn't port forwarding, it's getting IPV4 packets from the outside to your LAN over the external IPV6 link.
